src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -394,7 +394,7 @@ public void It_implicitly_defines_compilation_constants_for_the_target_framework
             definedConstants.Should().BeEquivalentTo(new[] { "DEBUG", "TRACE" }.Concat(expectedDefines).ToArray());
         }
 
-        [Theory]
+        [RequiresMSBuildVersionTheory("16.7.1")]
```

Comment:
I think this should just be `16.7`.  VS 16.7 hasn't released yet, and when it does I think the MSBuild version will be 16.7.0.something

Reply:
Or did microsoft/msbuild#5391 not go into VS 16.7?  In that case I think this value should be `16.8`.